pipeline/transform/transform.py

```diff
@@ -78,8 +86,9 @@ def clean_image_data(df:pd.DataFrame) -> pd.DataFrame:
 
 
 
+
 def format_watered_column(df:pd.DataFrame) -> pd.DataFrame:
-    """Converts the last_watered column to a datetime object with second precision."""
+    """Returns the last_watered column to a datetime object with second precision."""
```

Comment:
Re read this sentence; it doesn't quite make sense, perhaps you meant this?
```suggestion
    """Returns the datafram with the last_watered column converted to a datetime object with second precision."""
```